test/src/main/java/org/springframework/security/test/web/servlet/request/SecurityMockMvcRequestBuilders.java

```diff
@@ -124,6 +137,22 @@ public LogoutRequestBuilder logoutUrl(String logoutUrl, Object... uriVars) {
 
 		private LogoutRequestBuilder() {
 		}
+
+		@Override public boolean isMergeEnabled() {
+			return true;
+		}
+
+		@Override public Object merge( Object parent ) {
```

Comment:
Can you please put `@Override` on its own line and remove the extra space to be consistent with Spring Security code conventions?

```suggestion
		@Override
		public Object merge(Object parent ) {
```